### ISSUE TITLE
failed to deploy gcs

### ISSUE THREAD

**Issue by Madhu-1** [gluster/gcs#38]
Failed to Get peers in the cluster

```
Wednesday 17 October 2018  15:10:05 +0530 (0:00:00.864)       0:12:36.698 ***** 
changed: [kube1]

TASK [GCS | GD2 Cluster | Set gd2_client_endpoint] *****************************
Wednesday 17 October 2018  15:10:06 +0530 (0:00:00.533)       0:12:37.232 ***** 
ok: [kube1]

TASK [GCS | GD2 Cluster | Wait for glusterd2-cluster to become ready] **********
Wednesday 17 October 2018  15:10:06 +0530 (0:00:00.206)       0:12:37.439 ***** 
FAILED - RETRYING: GCS | GD2 Cluster | Wait for glusterd2-cluster to become ready (50 retries left).
FAILED - RETRYING: GCS | GD2 Cluster | Wait for glusterd2-cluster to become ready (49 retries left).
FAILED - RETRYING: GCS | GD2 Cluster | Wait for glusterd2-cluster to become ready (48 retries left).
FAILED - RETRYING: GCS | GD2 Cluster | Wait for glusterd2-cluster to become ready (47 retries left).
ok: [kube1]

TASK [GCS | GD2 Cluster | Get peers in cluster] ********************************
Wednesday 17 October 2018  15:10:52 +0530 (0:00:45.728)       0:13:23.168 ***** 
fatal: [kube1]: FAILED! => {"changed": false, "content": "", "msg": "Status code was -1 and not [200]: Request failed: <urlopen error [Errno 111] Connection refused>", "redirected": false, "status": -1, "url": "http://10.233.32.64:24007/v1/peers"}
```

**Comment by humblec** [comment]
@amarts @aravindavk @kshlm can some one take a look at this ?

**Comment by Madhu-1** [comment]
Fixed in #65
